plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -30,6 +30,7 @@ const NOT_SYNCED_PROPERTIES = new Set([
 export class TeamManager {
```

Comment:
We also already have an organizationmanager. Why put organization-level metrics into team manager?